cmd/fetch-from-syncada-via-sftp/main.go

```diff
@@ -139,26 +137,17 @@ func main() {
 	wrappedSFTPClient := invoice.NewSFTPClientWrapper(sftpClient)
 	syncadaSFTPSession := invoice.NewSyncadaSFTPReaderSession(wrappedSFTPClient, db, logger, v.GetBool(DeleteFilesFlag))
 
-	// Just use a processor that prints the files to stdout for now.
-	_, err = syncadaSFTPSession.FetchAndProcessSyncadaFiles(v.GetString(DirectoryFlag), t, &stdoutProcessor{})
+	_, err = syncadaSFTPSession.FetchAndProcessSyncadaFiles(v.GetString(DirectoryFlag), time.Time{}, invoice.NewEDI997Processor(db, logger))
```

Comment:
I think we should still be passing the parsed time param here instead of a zero time
```suggestion
	_, err = syncadaSFTPSession.FetchAndProcessSyncadaFiles(v.GetString(DirectoryFlag), t, invoice.NewEDI997Processor(db, logger))
```